src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+
+        Color randomColor = ColorSupplier.getRandomColor();
+        Random random = new Random();
+        int randomNumber = random.nextInt(100) + 1;
+        return new Ball(randomColor, randomNumber);
+    }
```

Comment:
let's not make this method static if there is no crucial need

Reply:
not fixed